lib/src/message_widget.dart

```diff
@@ -726,6 +731,7 @@ class _MessageWidgetState extends State<MessageWidget> {
                   widget.onReplyTap != null,
               showThreadReply:
                   widget.showThreadReplyIndicator && widget.onThreadTap != null,
+              showFlagButton: widget.showFlagButton && !isMyMessage,
```

Comment:
@deven98 why putting the condition `!isMyMessage` here if we put it in the messagelistview too?